sql-plugin/src/main/330+/scala/org/apache/spark/sql/execution/datasources/parquet/ShimCurrentBatchIterator.scala

```diff
@@ -82,8 +82,8 @@ class ShimCurrentBatchIterator(
 
   val sparkSchema = parquetColumn.sparkType.asInstanceOf[StructType]
   val parquetColumnVectors = (for (i <- 0 until sparkSchema.fields.length) yield {
-    new ParquetColumnVector(parquetColumn.children.apply(i),
-      vectors(i), capacity, MemoryMode.OFF_HEAP, missingColumns)
+    ParquetCVShims.newParquetCV(sparkSchema, i, parquetColumn.children.apply(i),
```

Comment:
why is there parquet changes in here?   If its fixing something else be sure to list it in the description of PR

Reply:
Added, it is fixing a build error.